apps/extension/src/shared/ui/components/select/select-option.component.tsx

```diff
@@ -13,12 +13,14 @@ export const SelectOption = forwardRef(
       <div
         ref={reference}
         className={classes(
-          'flex w-full cursor-pointer items-center space-x-2 border-none px-3 py-2 text-left',
+          'flex w-full cursor-pointer items-center space-x-2 border-none border-r border-r-neutral-200 px-3 py-2 text-left',
           'hover:bg-black/10 focus:bg-black/50',
           className,
         )}
       >
-        {option.prefix}
+       <div className='relative pr-2 mr-1 after:absolute after:right-0 after:-top-[6px] after:bg-gray-200 after:w-px after:h-[calc(2.625rem_-_6px)]'>
```

Comment:
please change the `after:-top-[6px]` to `after:top-[-6px]`

Reply:
I changed it to after:-top-1.5